ch_tools/common/clickhouse/config/clickhouse_keeper.py

```diff
@@ -38,6 +38,14 @@ def port_pair(self):
 
         return int(self._keeper_server.get("tcp_port", 0)), False
 
+    @property
+    def acl_username(self):
+        return self._keeper_server.get("acl_username")
```

Comment:
Does ClickHouse Keeper support this setting? Don't see it in https://clickhouse.com/docs/en/guides/sre/keeper/clickhouse-keeper#configuration

Reply:
No
I thought there was only one configuration file for this tool, but there is another one.